javascript/swiggy.js

```diff
@@ -0,0 +1,158 @@
+const pdpModule = (function() {
```

Comment:
This file should be named after the module it houses - rename it to `pdpModule.js`